tasks/proposals/setup-incentives.ts

```diff
@@ -0,0 +1,159 @@
+import { task } from "hardhat/config";
+import { exit } from "process";
+import { loadPoolConfig } from "../../helpers/market-config-helpers";
+import { MARKET_NAME } from "../../helpers/env";
+import { ZERO_ADDRESS } from "./../../helpers/constants";
+import { FORK } from "../../helpers/hardhat-config-helpers";
+import { addTransaction, getBatch } from "../../helpers/transaction-batch";
+import ProposalDecoder from "../../helpers/proposal-decoder";
+import { getEmissionManager, getAaveProtocolDataProvider } from "../../helpers/contract-getters";
+import { TransferStrategy, AssetType } from "./../../helpers/types";
+import { generateProposal } from "../../helpers/hydration-proposal.js";
+import { getPotRewardsStrategy } from "../../helpers/contract-getters";
+import { getBlockTimestamp } from "../../helpers/utilities/tx";
+import chalk from "chalk";
+
+task(`setup-incentives`, `Updates incentives program or starts new one if incentives doesn't exists.`)
+  .addParam("assets")
+  .setAction(async (
+      {
+        assets,
+      }: { assets: string },
+      hre
+    ) => {
+  const network = FORK ? FORK : (hre.network.name as eNetwork);
+
+  assets = assets.replaceAll(/\s/g, "").split(",")
+  if (assets.length == 0) {
+    console.log(chalk.red(`invalid '--assets' value, usage: --assets DOT,USDT`));
+    exit(1);
+  }
+
+  const poolConfig = await loadPoolConfig(MARKET_NAME);
+  const chainlinkConf = poolConfig.ChainlinkAggregator[network];
+  const incentivesConf = poolConfig.IncentivesConfig[network];
+  const em = await getEmissionManager();
+
+  if (!chainlinkConf) {
+    console.log(chalk.red(`chainlink configuration for ${network} network not found`));
+    exit(1);
+  }
+
+  const incentivizedTkns = Object.keys(incentivesConf);
+  const assetsConf = [];
+  const transferStrat = await getPotRewardsStrategy();
+  var emissionAdmin;
+
+  const dataProvider = await getAaveProtocolDataProvider();
+  const reserveTokens = await dataProvider.getAllReservesTokens();
+
+  for (let i = 0; i < assets.length; i++ ) {
+    const incTkn = assets[i];
+    const cfg = incentivesConf[incTkn];
+
+    if (!cfg) {
+      console.log(chalk.red(`Incentives config not found for asset: '${incTkn}'`));
```

Comment:
since you moved configuration into markets config, it would be better to have script that checks differences and makes sure to deploy / update it based on config. more akin to review-* scripts